static/static_file_modifier_test.go

```diff
@@ -222,4 +231,82 @@ func TestStaticModifierOnRequest(t *testing.T) {
 	if want := []byte("test file"); !bytes.Equal(got, want) {
 		t.Errorf("res.Body: got %q, want %q", got, want)
 	}
+
+	if got, want := res.ContentLength, int64(len("test file")); got != want {
+		t.Errorf("res.ContentLength: got %v, want %v", got, want)
+	}
+}
+
+func TestModifierFromJSON(t *testing.T) {
+	tmpdir, err := ioutil.TempDir("", "test_static_modifier_on_request_")
+	if err != nil {
+		t.Fatalf("ioutil.TempDir(): got %v, want no error", err)
+	}
+
+	if err := ioutil.WriteFile(path.Join(tmpdir, "sfmtest.txt"), []byte("test file"), 0777); err != nil {
+		t.Fatalf("ioutil.WriteFile(): got %v, want no error", err)
+	}
+
+	jsonStr := `{
+    "static.Modifier": {
+      "scope": ["request", "response"],
+      "rootPath": "<path>"
+    }
+  }`
+
+	msg := []byte(strings.Replace(jsonStr, "<path>", tmpdir, 1))
```

Comment:
Instead of this can you just do:
```go
msg := []byte(fmt.Sprintf(`{
  "static.Modifier": {
    "scope": ["request", "response"],
    "rootPath": %q
  }
}`, tmpdir))
```